{"name": "Принять заказ. Попытка принять заказ без id курьера возвращает корректное сообщение об ошибке", "status": "passed", "description": "тест ручки \"Принять заказ\"", "start": 1724853723785, "stop": 1724853723998, "uuid": "0cd9bf9d-3bd9-4cb5-9865-d0c3b0b9638d", "historyId": "bd10ff357b9f8b37e0339675f73a8320", "testCaseId": "bd10ff357b9f8b37e0339675f73a8320", "fullName": "tests.TestAcceptOrder#test_try_accept_order_without_courier_id", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "9332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}